xchange-stream-gateio/pom.xml

```diff
@@ -1,26 +1,110 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
     <parent>
-        <artifactId>xchange-parent</artifactId>
         <groupId>org.knowm.xchange</groupId>
+        <artifactId>xchange-parent</artifactId>
         <version>5.1.1-SNAPSHOT</version>
     </parent>
-    <modelVersion>4.0.0</modelVersion>
+    <artifactId>xchange-stream-gateio</artifactId>
 
     <name>XChange Gate.io Stream</name>
-    <artifactId>xchange-stream-gateio</artifactId>
+
+    <properties>
```

Comment:
versions should be defined in parent pom

Reply:
can do it, but nothing from the defined versions is used in the other modules

Reply:
why introduce these 4 dependencies anyway?

Reply:
- archunit - extends the tests, in these particular case ensures that there is no `Instant.now()` or `LocalDateTime.now()` calls in code. Only calls of now(Clock clock) are allowed. Otherwise impossible to test. See `ArchitectureTest`
- maven-enforcer-plugin - here makes sure that in final binary there are no dependencies with different versions, that can cause problems in runtime
- mockito - allows mocking during testing
- sortpom-maven-plugin - maintains the predefined structure of the pom file. Makes life way easier when merging the incoming changes.

None of these dependencies land in the final jar, they're used only during the tests.